release notes/v0.47.0.md

```diff
@@ -37,6 +37,16 @@ Thanks, @sapphire-janrain, for contribution!
 
 In some workflows the reflection call should also include some metadata. This PR adds [a new connection parameter `reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams) that allows to specify the metadata to be sent with the reflection call.
 
+### Improved the browser module's cookie API
+
+The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) that can be used while creating and retrieving cookies.
```

Comment:
Should the new feature go under the new feature section, and the rest of the changes go in the improvements section? In one line a reference to the improvements can be mentioned in the new section so readers know that there were improvements made to existing APIs.

Reply:
That makes sense. But, I want to give a comprehensive explanation of what we improved. Like we did [before here](https://github.com/grafana/k6/blob/39f0e6882228a061dbc495344dda5cc1261e7942/release%20notes/v0.46.0.md).